helm/stargate/templates/stargate_restapi_deployment.yaml

```diff
@@ -0,0 +1,51 @@
+{{ if .Values.restapi.enabled }}
```

Comment:
same comments as for the docs api

Reply:
Override of port number not needed to be in sync with stargate documentation.